Source/IntegrationTests/TestFiles/LitTests/LitTest/wishlist/GoModule.dfy

```diff
@@ -2,7 +2,7 @@
 // RUN: %exits-with 3 %dafny /compile:3 /unicodeChar:0 /spillTargetCode:2 "%s" /compileTarget:go 2> "%t"
 // note: putting /compileTarget:go after "%s" overrides user-provided option
 // RUN: %OutputCheck --file-to-check "%t" "%s"
-// CHECK: GoModuleConversions.go:10:3: "net/url" imported and not used
+// CHECK: undefined: GoModuleConversions.ParseURL
```

Comment:
Not clear on why this is the error we're getting now, but not a blocker since this is just a negative test.